recipes/jenkins.rb

```diff
@@ -207,12 +223,6 @@ def jenkins = Jenkins.getInstance()
     group 'jenkins'
   end
 
-  # Restart jenkins after updating the security realm otherwise running without
-  # authentication yields 403 errors when configuring.
-  service 'jenkins' do
-    action :restart
-  end
-
   # Aggregate permissions to assign to each user with a groovy script.
```

Comment:
This seems like treatment of a symptom rather than the underlying problem.

Changing one or both of TimeoutStopSec or RestartSec seems more appropriate since normal cookbook convergence shouldn't cause failures starting during configuration.